src/dev/pages/stack/stack.scss

```diff
@@ -0,0 +1,17 @@
+.stack-container {
+  height: 800px;
+  padding: 16px;
+  background: beige;
+  border-radius: 8px;
+}
+
+.box {
+    padding: 36px;
+    border: 4px dashed darkgray;
+    background-color: bisque;
```

Comment:
Super minor, but if you wanted to make this easier on the eyes in dark mode you could handle swapping these hard coded colors around depending on theme.